### PR TITLE
Added Atom Support #149

### PR DESCRIPTION
**Pull request by mattclements**
Refs #149

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
         "graham-campbell/throttle": "~2.0",
         "guzzlehttp/guzzle": "~5.0",
         "thujohn/rss": "~1.0",
-        "watson/validating": "0.10.*"
+        "watson/validating": "0.10.*",
+        "roumen/feed": "~2.8"
```
Comment (GrahamCampbell) [inline composer.json]: please order these alphabetically

File: app/config/app.php [hunk 2]
```diff
@@ -126,6 +126,7 @@
         'GrahamCampbell\Throttle\ThrottleServiceProvider',
         'GrahamCampbell\Markdown\MarkdownServiceProvider',
         'Thujohn\Rss\RssServiceProvider',
+        'Roumen\Feed\FeedServiceProvider',
```
Comment (GrahamCampbell) [inline app/config/app.php]: please put this above the rss provider

File: app/routes/app.php [hunk 3]
```diff
@@ -13,4 +13,5 @@
 
 Route::group(['namespace' => 'CachetHQ\Cachet\Http\Controllers'], function () {
     Route::get('/rss', 'RssController@feedAction');
+    Route::get('/atom', 'AtomController@feedAction');
```
Comment (GrahamCampbell) [inline app/routes/app.php]: please put this above the rss route

File: src/Http/Controllers/AtomController.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            } else {
+                $componentName = null;
+            }
+
+            $feed->add(
+                $incident->name,
+                Setting::get('app_name'),
+                Setting::get('app_domain'),
+                $incident->created_at,
+                ($componentName===null ? $incident->humanStatus : $componentName." ".$incident->humanStatus),
```
Comment (GrahamCampbell) [inline src/Http/Controllers/AtomController.php]: please put spaces around null

**Comment by mattclements** [comment]
Added Fixes to Styleguide

File: app/routes/app.php [hunk 5]
```diff
@@ -12,5 +12,6 @@
 });
 
 Route::group(['namespace' => 'CachetHQ\Cachet\Http\Controllers'], function () {
+	Route::get('/atom', 'AtomController@feedAction');
```
Comment (GrahamCampbell) [inline app/routes/app.php]: indent with spaces please

**Comment by GrahamCampbell** [comment]
Thank you @mattclements. :)

**Comment by mattclements** [comment]
No worries at all :+1:

**Pull request merged**